### PR TITLE
Add Kotlin as language option

### PR DESCRIPTION
**Pull request by bziemons**
Note: Makes LP dependent on [Forgelin](https://github.com/shadowfacts/Forgelin/)

### CODE REVIEW

**Comment by dblsaiko** [comment]
Kotlin, awesome!

By the way, you can make Drafts (I think they're called that?) instead of PRs that work the same but are unmergeable, so you don't have to put WIP in the title. [Here's a link](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
No idea if you can make existing PRs into drafts though.

**Comment by bziemons** [comment]
@therealfarfetchd oh, thanks for the hint :+1: didn't know that yet. I don't see a way to convert this PR after creation, but I will use drafts in the future :)

**Comment by bziemons** [comment]
@RS485/lp-devs @RS485/core please review :)

File: build.gradle [hunk 1]
```diff
@@ -115,10 +115,6 @@ repositories {
     maven {
         url "http://maven.shadowfacts.net/"
     }
-    maven {
```
Comment (dblsaiko) [inline build.gradle]: Maybe not remove it entirely and just comment it out, Player told me he'd fix it, he just can't do it right now. Not sure how long it'll take though

**Comment by bziemons** [comment]
I will strip the last commits from this PR later this evening.

File: build.gradle (hunk 1 above)
Comment (bziemons) [reply build.gradle]: I'd say if we "need" it again, we add it again. The maven cache I built has all our dependencies cached 30 days and acts as a proxy for any new artifacts.

In fact, gradle is built this way that if one repository does not answer at all, the build fails. The reason was that there are people with (imo nasty) setups that need reproducible builds. There is also no switch or config option to work around one repository timing out. That's why I built our cache and that's why I'd rather kick out any repository that fails sometimes.

**Pull request merged**